src/tests/pageContainer.test.js

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme'
-import PageContainer from '../containers/PageContainer'
-import SideBar from '../components/SideBar'
-import DashBoard from '../components/DashBoard'
+import { shallow, mount } from 'enzyme';
+import PageContainer from '../containers/PageContainer';
+import SideBar from '../components/SideBar';
+import DashBoard from '../components/DashBoard';
 import presetThemes from "../data/preset-themes";
 
 describe("PageContainer", () => {
   let wrapper;
 
   const aubergine = presetThemes[0].themeColor;
```

Comment:
Same comment here about decoupling the test from the data file, similar to `Dashboard.test.js` :)
You can imagine that this data would live in the separate db at some point.